dashboards_bundlers/extensionapp.py

```diff
@@ -59,6 +59,11 @@ def start(self):
                     'label': 'PHP Dashboard bundle (.zip)',
                     'module_name': 'dashboards_bundlers.php_download',
                     'group': 'download'
+                },
+                'dashboards_server_upload': {
+                    'label': 'Dashboard on Dashboard Server',
```

Comment:
Maybe Dashboard on Jupyter Dashboard Server just to give a bit more of a hint of where it's going?